aptos-move/framework/aptos-token/sources/token.move

```diff
@@ -1475,6 +1475,34 @@ module aptos_token::token {
         token_data.royalty
     }
 
+    /// return the token_data_id from the token_id
+    public fun get_tokendata_id(token_id: TokenId): TokenDataId {
+        token_id.token_data_id
+    }
+
+    /// return the mutation setting of the tokendata
```

Comment:
Can you update this comment to list the return values? Same for get_collection_mutability_config

Reply:
updated